cassandra/conf.yaml.example

```diff
@@ -157,170 +70,378 @@ init_config:
+              metric_type: counter
+              alias: jmx.gc.minor_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.minor_collection_time
+              metric_type: counter
+              alias: jmx.gc.minor_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: G1 Young Generation
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.minor_collection_count
+              metric_type: counter
+              alias: jmx.gc.minor_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.minor_collection_time
+              metric_type: counter
+              alias: jmx.gc.minor_collection_time
     # Old Gen Collectors (Major collections)
     - include:
        domain: java.lang
        type: GarbageCollector
        name: MarkSweepCompact
        attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: PS MarkSweep
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: ConcurrentMarkSweep
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
     - include:
         domain: java.lang
         type: GarbageCollector
         name: G1 Mixed Generation
         attribute:
           CollectionCount:
-            metric_type: counter
-            alias: jmx.gc.major_collection_count
+              metric_type: counter
+              alias: jmx.gc.major_collection_count
           CollectionTime:
-            metric_type: counter
-            alias: jmx.gc.major_collection_time
+              metric_type: counter
+              alias: jmx.gc.major_collection_time
```

Comment:
Here and for all the GC `metric_type` and `alias` above, there are two spaces too many for the indentation.
It seems to work anyway in this case but the indentation everywhere is two spaces so let's have that here also :)

Reply:
Makes sense, not sure what happened there 🙃. I am fixing it.